### ISSUE TITLE
Unit testing

### ISSUE THREAD

**Issue by ly29** [Sverchok/SverchokRedux#10]
It was always something we wanted to do to the old Sverchok but it was hard for series of reasons and we never did put the time into it.

But since we are starting over I think it also a good thing to consider.

For example:
- taking a node layout and duplicating and verifying the resulting layout dict matches, 
- some test layout is some different nodes against an expected result.
- node functions

Just running every node is worth a great deal as we have recently seen. I have no real experience how to set it up and this is a topic that will need some work, for now just raising the topic. Of course some issues with interaction with blender etc might no be possible to test.

https://docs.python.org/3/library/unittest.html

**Comment by zeffii** [comment]
Yes! It's worth trying to have a notion of testing in the back of the mind as these nodes start to become more numerous.

**Comment by ly29** [comment]
Exactly which is why it is important to start early
